include/exadg/time_integration/restart_data.h

```diff
@@ -60,11 +61,12 @@ struct RestartData
     }
   }
 
+  // TODO: use time_control
```

Comment:
the functionality is slightly different here. We also have the wall time in order to trigger restart. Hence, it isfine if we have a separate implementation here.